auto_users.h

```diff
@@ -0,0 +1,16 @@
+#ifndef AUTO_USERS_H
+#define AUTO_USERS_H
+
+extern char auto_usera[];
```

Comment:
all of them should be const.

Reply:
const does not appear in qmail-1.03.  I see we accidentally got one in the DESTDIR patch, and I see netqmail-1.04 added one in sendmail.c, but adding them, like adding ANSI function signatures, exceeds our mandate for the 1.07 release.

We should, for certain, add const systemically in the codebase--it's just not for this release.